src/components/Input/Input.st.css

```diff
@@ -0,0 +1,65 @@
+:import {
+    -st-from: "wix-ui-core/dist/src/components/Input/InputStyle.st.css";
+    -st-default: Input;
+  }
+  
+  :import {
+    -st-from: "wix-ui-core/dist/src/components/Input/Input.st.css";
+    -st-default: BaseInput;
+  }
+
+  :import {
+    -st-from: "../../palette.st.css";
+    -st-named: primaryText, mainHover, mainMutedHover, dividers;
+  }
+
+  :vars {
```

Comment:
Ideally this should be shared across all input-like components. And should also include input hover color based on palette instead of `dividers` var, even if it's the same color.